modules/apps/headless/headless-delivery/headless-delivery-impl/rest-openapi.yaml

```diff
@@ -9824,6 +9824,22 @@ paths:
                             schema:
                                 $ref: "#/components/schemas/MessageBoardMessage"
             tags: ["MessageBoardMessage"]
+    "/message-board-messages/{messageBoardMessageId}/mark-as-answer":
+        put:
```

Comment:
A question about the semantics: shouldn't this use POST? This is not putting any subresource, it asking for some processing of the associated message.

Reply:
I followed the pattern of the subscribe/unsubscribe action pairs, where an operation on an entry is performed.
https://github.com/liferay/liferay-portal/blob/master/modules/apps/headless/headless-delivery/headless-delivery-impl/rest-openapi.yaml#L10056-L10057

The POST requests defined in rest-openapi.yaml are for creating entries, for example:
https://github.com/liferay/liferay-portal/blob/master/modules/apps/headless/headless-delivery/headless-delivery-impl/rest-openapi.yaml#L11380
https://github.com/liferay/liferay-portal/blob/master/modules/apps/headless/headless-delivery/headless-delivery-impl/rest-openapi.yaml#L12253

Reply:
Ok, if this is an established pattern them I'm fine with it.